query/stdlib/influxdata/influxdb/from.go

```diff
@@ -3,23 +3,576 @@ package influxdb
 import (
 	"fmt"
 
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
 	"github.com/influxdata/flux/execute"
 	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/semantic"
 	"github.com/influxdata/flux/stdlib/influxdata/influxdb"
+	"github.com/influxdata/flux/stdlib/universe"
+	"github.com/influxdata/flux/values"
 	platform "github.com/influxdata/influxdb"
 	"github.com/influxdata/influxdb/query"
 	"github.com/pkg/errors"
 )
 
+const FromKind = "influxDBFrom"
```

Comment:
Does the `influxdb.from` need a different kind than the one in vanilla Flux?  This seems okay, I'm just curious.

Reply:
Yep, it has to, otherwise, when registering the opSpec it would result in an error. Kinds must be unique, while package values can be replaced.